site/src/content/docs/ref/init-package.mdx

```diff
@@ -104,6 +104,18 @@ The `registry:2` image and the Zarf Agent image can be configured with a custom
 
 :::
 
+## Cluster creation
+
+If there is no running cluster, the init package can be used to create one. It has a deployable K3s cluster component that can be optionally deployed on your machine. To install a K8s distribution other than k3s, finding or creating a custom Zarf package is necessary.
+
+Typically, an init package is the first Zarf Package to be deployed on a cluster as other packages often rely on the services it installs or configures, however Zarf may be used to package and install the cluster distribution itself.  In this scenario, the distribution package would be deployed first, followed by the init package, and then any other packages planned for deployment.
+
+:::tip
+
+Check out our [k3s cluster package](https://github.com/defenseunicorns/zarf/blob/main/packages/distros/k3s/zarf.yaml) to see an example of a Zarf package which installs a kubernetes distribution
+
+:::
```

Comment:
I don't find this section very helpful because we already describe the `k3s` component in the optional components table in this same doc page

Reply:
The message I want to get across here  is that Zarf can be used to create clusters which I think is a relatively use common case for the air gap. Maybe there's a better place for it? Currently this block is actually both in this page-> https://docs.zarf.dev/ref/create/#zarfinitconfig and this page-> https://docs.zarf.dev/ref/packages/#zarfinitconfig though I reworded & shortened it a bit

Reply:
Ended up moving this back to the create.mdx file and bringing that file back